diff --git a/inflection/__init__.py b/inflection/__init__.py
--- a/inflection/__init__.py
+++ b/inflection/__init__.py
@@ -225,4 +225,4 @@ def ordinal(number: int) -> str:
     number = abs(int(number))
-    if number % 100 in (11, 12):
+    if number % 100 in (11, 12, 13):
         return "th"
     else:
@@ -274,6 +274,6 @@ def parameterize(string: str, separator: str = '-') -> str:
     if separator:
         re_sep = re.escape(separator)
         # No more than one of the separator in a row.
-        string = re.sub(r'%s{1,}' % re_sep, separator, string)
+        string = re.sub(r'%s{2,}' % re_sep, separator, string)
         # Remove leading/trailing separator.
         string = re.sub(r"(?i)^{sep}|{sep}$".format(sep=re_sep), '', string)
